src/operations.js

```diff
@@ -0,0 +1,87 @@
+import { curry } from './functions'
+import { affineFold } from './AffineFold'
+import { affineTraversal } from './AffineTraversal'
+import { getter } from './Getter'
+import { lens } from './Lens'
+import { prism } from './Prism'
+import { reviewer } from './Reviewer'
+import { setter } from './Setter'
+
+// iso, /prism, /lens,
+// /affinetraversal, /getter,
+// /affinefold, traversal,
+// /reviewer, fold, /setter
+const ocompose2 = (optic1, optic2) => {
```

Comment:
What if you call this `composeOptics`? I think the name is better 😂 

Also, why not do like monocle-ts and compose each optic from their prototype??

```js
Lens.fromPop('foo').composePrism('bar')
```

Reply:
Sure! I've changed `ocompose` to `composeOptics`.

I really think that the style using `composeOptics` or `path` looks nicer. And we do not have to add every other single possible way to create an optic to each prototype.